packages/application-generic/src/health/queue-health-indicator.service.ts

```diff
@@ -5,7 +5,7 @@ import {
 } from '@nestjs/terminus';
 import { Injectable, Logger } from '@nestjs/common';
 
-import { QueueBaseService } from '../services';
+import { QueueBaseService } from '../services/queues/queue-base.service';
```

Comment:
There is a circular dependency when executing the App Generic tests for the Queues and Workers. Still haven't been able to find the guilty one. These import path changes are part of that investigation.